src/plone/volto/behaviors/preview_link.py

```diff
@@ -56,16 +56,17 @@ def __init__(self, field, context, request):
 
     def __call__(self):
         value = self.field.get(self.context)
-        linked_image = value.to_object
-        primary_field = IPrimaryFieldInfo(linked_image).field
-        serializer = queryMultiAdapter(
-            (primary_field, linked_image, self.request), IImageScalesFieldAdapter
-        )
-        if serializer is not None:
-            values = serializer()
-            if values:
-                portal_url = api.portal.get().absolute_url()
-                base_path = linked_image.absolute_url().replace(portal_url, "")
-                for value in values:
-                    value["base_path"] = base_path
-            return values
+        if value:
```

Comment:
Can you make it return an empty list if there's no value, so it's the same shape as when there is an image?

Reply:
Sure! I had doubts on this one. Done!